components/proxy/src/test/kotlin/com/hotels/styx/services/OriginsAdminHandlerTest.kt

```diff
@@ -167,10 +171,21 @@ class OriginsAdminHandlerTest : FeatureSpec({
             expectStateChange(STATE_UNREACHABLE, STATE_ACTIVE, STATE_UNREACHABLE, false)
         }
 
-        scenario("Activating an inactive origin results in an active state") {
+        // TODO: Complete this test
+        scenario("Activating an unhealthchecked, inactive origin results in an active state") {
             expectStateChange(STATE_INACTIVE, STATE_ACTIVE, STATE_ACTIVE, false)
         }
 
+        // TODO: Complete this test
+        scenario("Activating a healthchecked, inactive origin results in an unreachable state") {
+            expectStateChange(STATE_INACTIVE, STATE_ACTIVE, STATE_UNREACHABLE, false, "app.origin.hc.running")
+        }
+
+        // TODO: Complete this test
```

Comment:
3xTODOs here. Are these still work in progress? Or just old comments?

Reply:
Ah, these are old. Will remove.